### PR TITLE
worked on video challenge superblock

### LINKED ISSUE #6628: Display video challenge superblock in challenge map

### PR DESCRIPTION
**Pull request by AkiraLaine**
closes #6628 (thanks to @SaintPeter for helping :smile:)

fixed rebase merge errors

Fix videos route

Fix hikes broken true/false

### CODE REVIEW

**Comment by SaintPeter** [comment]
I checked this out and everything appears to be working.

There is one funny thing I see in the local console:
> Warning: Failed propType: Invalid prop `id` of type `number` supplied to `Lecture`, expected `string`. Check the render method of `LectureContainer`.

It doesn't seem to affect the function of the video challenges, though.

@BerkeleyTrue I suggest we put this up on Beta (if you're done with it) for testing.

**Comment by BerkeleyTrue** [comment]
This can be tested locally.

**Comment by SaintPeter** [comment]
@FreeCodeCamp/issue-moderators 
We need some more QA on this one, then.

**Comment by BerkeleyTrue** [comment]
@AkiraLaine Please rebase this PR off staging

File: common/app/routes/Hikes/components/Hike.jsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ export default contain(
     actions: ['hikesActions']
   },
   React.createClass({
-    displayName: 'Hike',
+    displayName: 'Video',
```
Comment (BerkeleyTrue) [inline common/app/routes/Hikes/components/Hike.jsx]: This PR should not change the internal names of components, just the user facing names.

Comment (SaintPeter) [inline common/app/routes/Hikes/components/Hike.jsx]: Sorry, I thought this was what it say:  a "displayName", not a component name. I'll fix it.

**Comment by AkiraLaine** [comment]
All fixed, tested locally - everything seems to work

**Comment by SaintPeter** [comment]
I think this is ready to merge.

@ltegman Have you tested?

**Comment by QuincyLarson** [comment]
Great work!

File: server/boot/a-react.js [hunk 2]
```diff
@@ -15,8 +15,8 @@ const debug = debugFactory('freecc:react-server');
 const routes = [
   '/jobs',
   '/jobs/*',
-  '/hikes',
-  '/hikes/*'
+  '/videos',
+  '/videos/*'
 ];
 
 const devRoutes = [];
```
Comment (Cade9) [inline server/boot/a-react.js]: Riggt

File: common/app/routes/Hikes/index.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ import Hike from './components/Hike.jsx';
  */
 
 export default {
-  path: 'hikes',
+  path: 'videos',
   component: Hikes,
   childRoutes: [{
     path: ':dashedName',
```
Comment (Cade9) [inline common/app/routes/Hikes/index.js]: Fwm

**Pull request merged**